cmd/nodepool/aws/log.go

```diff
@@ -0,0 +1,7 @@
+package aws
```

Comment:
I'm not sure where this pattern of having a single file for this variable is coming from?
I question we should keep perpetuating it? may @ironcladlou knows better where this is coming from/why we need it?

Reply:
I think it's just a combination of organic changes and cargo culting. Quick analysis:

In the CLI we're relying on a package variable for the logging singleton (`util.Log`). Authors might use `util.Log.Error(...)` (for example) directly. Or alias it to a local so it would read more like `log.Error(...)` (what this PR is doing).

If we're okay fundamentally with the package variable for logger state (instead of e.g. passing logger instances around), we could move the package variable to a `log` package so that users get the concise `log.Error(...)` call semantics without doing aliasing.

Reply:
I agree with the last suggestion
I can create new log package under cmd, to be used from all cmd sub packages

Reply:
Changed it
Need to change also the other packages to use this new log package (another PR?)